py3status/modules/spaceapi.py

```diff
@@ -57,10 +52,9 @@ def check(self):
                 self.closed_color = ''
 
             # grab json file
-            json_file = urllib.request.urlopen(self.url)
-            reader = codecs.getreader("utf-8")
-            data = json.load(reader(json_file))
-            json_file.close()
+            data = self.py3.get_json_data(self.url)
+            if data is None:
+                self.py3.notify_user('{} cannot be reached.'.format(self.url))
```

Comment:
imho I think we shouldn't notify the user like this.  eg if there is no network.  We can either show them cached info on the i3bar or a 'unavailable' message.

Before it just crashed so anything is an improvement.